projects/gopacket/project.yaml

```diff
@@ -0,0 +1,7 @@
+homepage: "https://github.com/google/gopacket"
+primary_contact: "p.antoine@catenacyber.fr"
+
```

Comment:
Can you also CC project maintainer(s) on this PR? We need them to review the fuzz target + someone else has to be the primary contact / auto_ccs

Reply:
I will do a PR for the fuzz target and see who answers